vertx/feign-vertx/src/main/java/feign/VertxFeign.java

```diff
@@ -122,13 +120,13 @@ public Builder invocationHandlerFactory(
     }
 
     /**
-     * Sets a vertx instance to use to make the client.
+     * Sets a vertx WebClient.
      *
-     * @param vertx vertx instance
+     * @param webClient vertx WebClient
      * @return this builder
      */
-    public Builder vertx(final Vertx vertx) {
```

Comment:
My main concern about this change is breaking compatibility with previous version.


Would it be possible to keep this (probably as deprecate) and run `WebClient.create(vertx)`

Then we can instruct to use the new method and mark for removal on next major release

Reply:
No need for that. Because we never released the current version. The previous is still available, by the way, in the separate project that was merged into the main project recently.